src/main/java/com/mraof/minestuck/blockentity/ComputerBlockEntity.java

```diff
@@ -221,6 +242,34 @@ public void putServerMessage(String message)
 		markBlockForUpdate();
 	}
 	
+	public void burnDisk(int programId)
+	{
+		if(level == null)
+			return;
+		
+		BlockPos bePos = getBlockPos();
+		ItemStack diskStack = ProgramData.getItem(programId);
+		if(!diskStack.isEmpty() && blankDisksStored > 0)
```

Comment:
Realized that you never actually verify here that this computer has the info to burn these disks. You need to do this.

Reply:
The button is not displayed on the screen if the computer does not have the necessary components which keeps it in check, or is there another angle to the issue?

Reply:
Yeah, there's network packet security. A player with a modified mod (or perhaps just a custom addon) could in theory send whatever packet they like. For that reason, when a player says that it performs some action by sending a packet, basic level of anti-cheat is to check that the player actually can perform that action.